docs/examples/gestures/lib/main_multitap.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+import 'package:flame/gestures.dart';
+
+void main() {
+  final game = MyGame();
+  runApp(game.widget);
+}
+
+/// Includes an example including advanced detectors
+class MyGame extends Game with MultiTouchTapDetector {
+  final _whitePaint = Paint()..color = const Color(0xFFFFFFFF);
```

Comment:
you can grab white from the basic palette 

final _whitePaint = BasicPalette.white.paint;